World.py

```diff
@@ -55,6 +55,9 @@ def __init__(self, id, settings, resolveRandomizedSettings=True):
 
         self.shuffle_special_interior_entrances = settings.shuffle_interior_entrances == 'all'
         self.shuffle_interior_entrances = settings.shuffle_interior_entrances in ['simple', 'all']
+		
+        self.shuffle_special_dungeon_entrances = settings.shuffle_dungeon_entrances == 'all'
+        self.shuffle_dungeon_entrances = settings.shuffle_dungeon_entrances in ['simple', 'all']
```

Comment:
The usage of `World.settings.shuffle_dungeon_entrances` in the remaining codebase should be replaced with `World.shuffle_dungeon_entrances` accordingly.